### ISSUE TITLE
Move alertgroup to an other team

### ISSUE THREAD

**Issue by ToonTijtgat2** [grafana/oncall#2433]
### What would you like to see!

Add the possibility to move an alert with a resolution note to an other team.
This could be usefull if you rely on an other team to solve this alert.

### Product Area

Alert Flow & Configuration

**Comment by KevinDW-Fluxys** [comment]
I agree.  We have the use case of a first line team who can handle some incoming alerts, but will forward others. They can now add the target team as a responder via their schedule, but the alert group remains assigned to their team. 
This means it only appears in their alert group overview. Having the alert in the correct teams overview would be a nice addition, as well as having a clear view on which team is handling this alert.
